storage/aptosdb/src/db_options.rs

```diff
@@ -33,6 +33,46 @@ pub(super) fn ledger_db_column_families() -> Vec<ColumnFamilyName> {
     ]
 }
 
+pub(super) fn event_db_column_families() -> Vec<ColumnFamilyName> {
+    vec![
+        /* empty cf */ DEFAULT_COLUMN_FAMILY_NAME,
+        DB_METADATA_CF_NAME,
+        EVENT_ACCUMULATOR_CF_NAME,
+        EVENT_BY_KEY_CF_NAME,
+        EVENT_BY_VERSION_CF_NAME,
+        EVENT_CF_NAME,
+    ]
+}
+
+pub(super) fn transaction_db_column_families() -> Vec<ColumnFamilyName> {
+    vec![
+        /* empty cf */ DEFAULT_COLUMN_FAMILY_NAME,
+        DB_METADATA_CF_NAME,
+        TRANSACTION_CF_NAME,
+        TRANSACTION_BY_ACCOUNT_CF_NAME,
+        TRANSACTION_BY_HASH_CF_NAME,
+    ]
+}
+
+pub(super) fn write_set_db_column_families() -> Vec<ColumnFamilyName> {
+    vec![
+        /* empty cf */ DEFAULT_COLUMN_FAMILY_NAME,
+        DB_METADATA_CF_NAME,
+        WRITE_SET_CF_NAME,
+    ]
+}
+
+pub(super) fn ledger_metadata_db_column_families() -> Vec<ColumnFamilyName> {
+    vec![
+        /* empty cf */ DEFAULT_COLUMN_FAMILY_NAME,
+        DB_METADATA_CF_NAME,
```

Comment:
why does this CF exists in write_set_db_column_families as well?

Reply:
Every db keeps its own metadata, to record some information like commit progress and pruner progress.